.env.test

```diff
@@ -1,4 +1,4 @@
 # define your env variables for the test env here
 KERNEL_CLASS='App\Kernel'
 APP_SECRET='s$cretf0rt3st'
-DATABASE_URL=mysql://db_user:db_password@127.0.0.1:3306/db_name_test?serverVersion=5.7
```

Comment:
this is now set in the `config/packages/test/doctrine.yaml` via a `_test` prefix

Reply:
strange 🙈  but OK for me